src/__Private/LinterCLIConfig.php

```diff
@@ -146,27 +156,33 @@ public function getConfigForFile(string $file_path): self::TFileConfig {
         continue;
       }
 
-      $linters = Vec\concat(
-        $linters,
-        $override['extraLinters'] ?? vec[],
-      );
-      $blacklist = Vec\concat(
-        $blacklist,
-        $override['disabledLinters'] ?? vec[],
-      );
-      $autofix_blacklist = Vec\concat(
-        $autofix_blacklist,
-        $override['disabledAutoFixes'] ?? vec[],
-      );
-      $no_autofixes = $no_autofixes ||
-        ($override['disableAllAutoFixes'] ?? false);
+      if ($override['disableAllLinters'] ?? false) {
+        $linters = $normalize(vec[]);
+        $autofix_blacklist = $normalize(vec[]);
+        $linters = $assert_types($linters);
+        $autofix_blacklist = $assert_types($autofix_blacklist);
```

Comment:
The noramlize/assert_types lambdas seems unnecessary  - this is equivalent to:

```
return shape(
  'linters' => keyset[],
  'autoFixBlacklist' => keyset[],
);
```

also, setting the autofixblacklist should be unneccessary if we're not running any linters

Reply:
ah, i was looking for this.